dev-tools/mage/dockerbuilder.go

```diff
@@ -46,15 +46,15 @@ func newDockerBuilder(spec PackageSpec) (*dockerBuilder, error) {
 
 func (b *dockerBuilder) Build() error {
 	if err := os.RemoveAll(b.buildDir); err != nil {
-		return errors.Wrapf(err, "failed to clean existing build directory %s", b.buildDir)
+		return fmt.Errorf("failed to clean existing build directory %s: %w", b.buildDir, err)
```

Comment:
It's probably worth someone to look over these changes from `pkg/errors` (due to linting complaints) to `fmt.Errorf` calls to make sure I did them correctly.